x-pack/legacy/plugins/siem/public/components/ml/tables/anomalies_network_table.tsx

```diff
@@ -57,33 +54,29 @@ export const AnomaliesNetworkTable = React.memo<AnomaliesNetworkTableProps>(
       hidePerPageOptions: false,
     };
 
-    if (!hasMlUserPermissions(capabilities)) {
-      return null;
-    } else {
-      return (
-        <Panel loading={{ loading }}>
-          <HeaderPanel
-            subtitle={`${i18n.SHOWING}: ${pagination.totalItemCount.toLocaleString()} ${i18n.UNIT(
-              pagination.totalItemCount
-            )}`}
-            title={i18n.ANOMALIES}
-            tooltip={i18n.TOOLTIP}
-          />
+    return (
```

Comment:
Would like to know if there's a specific reason that we are removing the permission check here?